src/lucky_web/tags/text_helpers.cr

```diff
@@ -0,0 +1,217 @@
+module LuckyWeb::TextHelpers
+  @@_cycles = Hash(String, Cycle).new
+
+  def truncate(text, length = 30, omission = "...", separator = nil, escape = false, blk : Nil | Proc = nil)
+    if text
```

Comment:
I think we should disallow nil text and only allow String instead.

Reply:
I'm not attached to the nil check. There are specific test cases in Rails for a reason and my guess is nilable text from DB records.